config/initializers/assets.rb

```diff
@@ -12,3 +12,5 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( show.css )
```

Comment:
You don't need to add this line if you import `show` and `index` in `application.scss`

Reply:
Wouldn't importing those files in application.scss also apply the styles from index.css and show.css onto the application.html page?

Reply:
Yeah they'll apply to everything, which in general is ok since a style in `show` shouldn't break the style in `index`. 

However, there are ways of having different stylesheets for specific controllers but that requires a little bit more work that I don't think is necessary. Here's a tutorial on how to have page specific javascript (the process will be the exact same but with css) http://blog.seancarpenter.net/2012/11/05/page-specific-javascript-with-the-asset-pipeline/

Reply:
Is there an action that needs to be taken here?

Reply:
I think we should let Rails do the assets autoloading magic instead of handling it ourselves so I'm game for what the link describes 👍 

Since this is a prototype project, I'm not too concerned with loading all the css in every file. If we really wanted to get fancy with it, we could use the SMACSS pattern but I really don't think it's necessary